readme.md

```diff
@@ -13,19 +13,14 @@ $ npm install first-chunk-stream
 ## Usage
 
 ```js
-const fs = require('fs');
-const getStream = require('get-stream');
-const FirstChunkStream = require('first-chunk-stream');
+import * as fs from 'fs';
+import getStream = require('get-stream');
+import FirstChunkStream = require('first-chunk-stream');
```

Comment:
This is not correct. It should stay CommonJS-style.

Reply:
Yeah, sorry, copypasted from`index.d.ts` >_<